hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -3257,19 +2948,7 @@ public MerkleTreeConfig findMapMerkleTreeConfig(String name) {
      * @see #getConfigPatternMatcher()
      */
     public MerkleTreeConfig getMapMerkleTreeConfig(String name) {
-        name = getBaseName(name);
-        MerkleTreeConfig config = lookupByPattern(configPatternMatcher, mapMerkleTreeConfigs, name);
-        if (config != null) {
-            return config;
-        }
-        MerkleTreeConfig defConfig = mapMerkleTreeConfigs.get("default");
-        if (defConfig == null) {
-            defConfig = new MerkleTreeConfig().setMapName("default").setEnabled(false);
-            addMerkleTreeConfig(defConfig);
-        }
-        config = new MerkleTreeConfig(defConfig).setMapName(name);
-        addMerkleTreeConfig(config);
-        return config;
+        return ConfigUtils.getConfig(configPatternMatcher, mapMerkleTreeConfigs, name, MerkleTreeConfig.class);
```

Comment:
The merkle tree config is similar to the event journal config. It needs to be disabled if the name is "default".

Reply:
I did not notice that. I will fix it.